doc/source/tutorial/linalg_batch.md

```diff
@@ -0,0 +1,148 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+orphan: true
+---
+
+(linalg_batch)=
```

Comment:
Yes, this is equivalent to `.. linalg_batch::` in rst. I did not create it initially because I was able to use the `:doc:` reference when building locally (and doc takes a document file name without extension).

Reply:
Right - I tried `:doc:` in the previous commit (63ab46f), and it works from the tutorial `.rst`, but not from the docstring of `eig`. Is it supposed to be that way, perhaps because it's easier to resolve the file name from within the `doc` folder?